src/components/ProductCardVertical.jsx

```diff
@@ -1,25 +1,83 @@
-import { useContext, useState } from "react";
-import { Link } from "react-router-dom";
-import { CartContext } from "../contexts/cart-context";
+import { Link, useNavigate } from "react-router-dom";
+import { useCart } from "../contexts/cart-context";
+import { toast } from "react-toastify";
 import "./ProductCardVertical.css";
+import { useAuth } from "../contexts/auth-context";
+import {
+  addItemToCartApi,
+  addToWishlistApi,
+  decreaseItemFromCartApi,
+} from "../apis";
+import { useState } from "react";
+import DecreaseItemQtyButton from "./Buttons/DecreaseItemQtyButton";
+import IncreaseItemQtyButton from "./Buttons/IncreaseItemQtyButton";
+import AddToCartButton from "./Buttons/AddToCartButton";
```

Comment:
suggestion: You can import all the button components into an index file within Buttons folder and import all the components from that file in a single line.

Reply:
Nice suggestion, will look into this.